sdk/core/azure-core-amqp/src/amqp/claim_based_security.cpp

```diff
@@ -68,21 +64,21 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
       auto rv{m_management->Open(context)};
       switch (rv)
       {
-        case ManagementOpenStatus::Invalid:
-          return CbsOpenResult::Invalid;
+        case ManagementOpenStatus::Invalid: // LCOV_EXCL_LINE
```

Comment:
There are many, many cases where it's impossible for us to hit the failure without a significant investment in mocking. This is especially true for the CI tests since they don't deal with real-world networking issues.

I can remove all the coverage exclusions and let the chips fall but our numbers will be dramatically lower.